lib/utilities/is-project.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = function isProject(obj) {
+  return obj.constructor && obj.constructor.name === 'Project';
```

Comment:
I do want to change this to something else (not relying on Function.name), _maybe_ using:

https://github.com/ember-cli/ember-cli/blob/b24b73b388934796ca915ca665b48a27c857199b/lib/models/project.js#L195

Or `obj instanceof Project`?

Reply:
Done - I check isEmberCliProject() after testing for the symbol being present.